tests/Unit/view/LoginViewTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Unit\view;
```

Comment:
```suggestion
namespace Tests\Unit\View;
```
m bảo sử dụng Laravel Dusk cơ mà :(